src/plugin-store.svelte

```diff
@@ -74,8 +74,20 @@ $: editorPlugins = filteredPlugins.filter((it) => it.kind === "editor");
 $: menuPlugins = filteredPlugins.filter((it) => it.kind === "menu");
 $: validatorPlugins = filteredPlugins.filter((it) => it.kind === "validator");
 
+//#region UI
+
 let pluginStore: Element;
 
+function openPluginDownloadUI() {
+    document.dispatchEvent(new Event("open-plugin-download"));
+}
+
+document.addEventListener("add-external-plugin", () => {
```

Comment:
we could use svelte's special elements to add events to document:
https://svelte.dev/docs/svelte/svelte-document